content/concepts/payment-system-basics/accounts/accounts.md

```diff
@@ -75,6 +75,8 @@ To be deleted, an account must meet the following requirements:
 
 After an account has been deleted, it can be re-created in the ledger through the normal method of [creating accounts](#creating-accounts). An account that has been deleted and re-created is no different than an account that has been created for the first time.
 
+**Warning:** The [AccountDelete transaction][]'s transaction cost applies even if the transaction fails because the account does not meet the requirements to be deleted. To avoid paying 5 XRP in case account deletion fails, [submit the transaction](submit.html) with `fail_hard` enabled so that the transaction is rejected unless it is provisionally successful.
```

Comment:
Suggestion:

**Warning**: The [AccountDelete transaction][]'s cost applies if the transaction is properly structured and signed, but the account does not meet the requirements to be deleted. [Submit the transaction](submit.html) with `fail_hard` enabled to lower the chances of paying the high transaction fee if the account is not deletable when the transaction is validated.